features/object_mapper/creating/README.md

```diff
@@ -0,0 +1,168 @@
+## Definition of mapped classes
+
+The object mapper allows to define mapped classes by using annotations.
+ 
+### Creating a table entity
+ 
+A table entity is generally meant to match the definition of an existing table in the *Cassandra* server.
```

Comment:
"Generally" seems to imply that there are exceptions, were you thinking about anything in particular (like the possibility of not mapping all columns in a table)?

On a side note, the use of italics for Cassandra looks a bit strange to me.